src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxItem.cs

```diff
@@ -454,12 +441,12 @@ protected virtual void Deserialize(SerializationInfo info, StreamingContext cont
                 // names we use
                 propertyNames = new string[]
                 {
```

Comment:
All serialization related code paths should be deleted for now.

Reply:
Sure. I need to discuss with team and you on serialization story for winforms in general.

Reply:
Also, do we allow commented out source code in corefx? or it should be completely removed? In case if we revisit to enable it, it might be easier.